### ISSUE TITLE
Ability to Return Issued Items

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#1003]
## Precondition

- Returns must be initiated *from the Finance System* and can only be processed when a transaction is an `Issued` state.

- Also, a single transaction may only have one return. In order to do a second return, the admin staff would have to ask for help from DTS. 

## Test Script

1. Start from an Inventory Request has an inventory item in a status of `Issued`

2. In the rightmost column of the table (`Return`), you should see an icon with a `Return` icon. Click the icon, then `Submit` the form that appears.

### Expected behavior

- A new transaction will be inserted on the request, with a status of `Returned` and a quantiy which matches the `Return Quantity` on the submitted form.

- In the Data Tracker, a `Return` transaction will be added to the work order matching the transaction in the finance system.

**Comment by johnclary** [comment]
@dianamartin ready for testing

**Comment by dianamartin** [comment]
Used Inventory Request `72`. This is in a `Needs AIMS entry` status for the Inventory Request. 
The inventory item status is `Issued`. 

I clicked the `Return` button. 
I had the ability to indicate what quantity is to be returned and who returned the item. 

A new record was added to the Inventory Item Request(s) table, it was a record in a `Returned` status. It shows the quantity I entered to be returned and who returned the item. 

Checked the `Transactions` table and saw the returned transaction there.
